.github/workflows/release.yml

```diff
@@ -315,3 +391,22 @@ jobs:
         run: |
           MANIFEST=$(aws ecr batch-get-image --repository-name covalent --image-ids imageTag=${{ github.events.inputs.stable_version }} --output json | jq --raw-output --join-output '.images[0].imageManifest')
           aws ecr put-image --repository-name covalent --image-tag stable --image-manifest "$MANIFEST"
+
+      - name: Build AWS executor table image
+        if: >
+          !inputs.prerelease
+          && github.events.inputs.stable_version
+          && !github.events.inputs.test_release
+          && contains(env.PAUL_BLART, github.actor)
+        uses: docker/build-push-action@v2
+        with:
+          builder: ${{ steps.buildx.outputs.name }}
+          context: .
+          file: ./Dockerfile.awsexecutor
+          platforms: [linux/amd64, linux/arm64]
+          build-args:
+            - COVALENT_BASE_IMAGE: public.ecr.aws/covalent/covalent:stable
+          push: true
+          tags: |
+            public.ecr.aws/covalent-executor-base/covalent:latest
```

Comment:
Remove the `latest` from this step, since this will usually be _behind_ the `latest` image.

Reply:
I'm pretty sure `latest` should be on the prerelease? @wjcunningham7 can check me on that